plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/revisionsList/RevisionListViewImpl.java

```diff
@@ -0,0 +1,214 @@
+    @Override
+    public void setDescription(String description) {
+        this.description.setText(description);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void close() {
+        this.hide();
+        onClose();
+    }
+
+    @Override
+    protected void onClose() {
+        selectionModel.clear();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void showDialog() {
+        this.show();
+    }
+
+    private void createRevisionsTable(org.eclipse.che.ide.Resources coreRes) {
+        Column<Revision, String> idColumn = new Column<Revision, String>(new TextCell()) {
+            @Override
+            public String getValue(Revision revision) {
+                return revision.getId().substring(0, 8) + "...";
+            }
+        };
+        Column<Revision, String> timeColumn = new Column<Revision, String>(new TextCell()) {
+            @Override
+            public String getValue(Revision revision) {
+                return dateTimeFormatter.getFormattedDate(revision.getCommitTime());
+            }
+        };
+        Column<Revision, String> authorColumn = new Column<Revision, String>(new TextCell()) {
+            @Override
+            public String getValue(Revision revision) {
+                return revision.getCommitter().getName();
+            }
+        };
+        Column<Revision, String> titleColumn = new Column<Revision, String>(new TextCell()) {
+            @Override
+            public String getValue(Revision revision) {
+                return revision.getMessage().substring(0, 50);
+            }
+        };
+
+        revisions = new CellTable<>(15, coreRes);
+
+        revisions.setWidth("100%");
+
+        revisions.addColumn(idColumn, locale.viewCompareRevisionTableIdTitle());
+        revisions.addColumn(timeColumn, locale.viewCompareRevisionTableTimeTitle());
+        revisions.addColumn(authorColumn, locale.viewCompareRevisionTableAuthorTitle());
+        revisions.addColumn(titleColumn, locale.viewCompareRevisionTableTitleTitle());
+
+        selectionModel = new SingleSelectionModel<Revision>();
+        selectionModel.addSelectionChangeHandler(new SelectionChangeEvent.Handler() {
+            @Override
+            public void onSelectionChange(SelectionChangeEvent event) {
+                description.setText(selectionModel.getSelectedObject().getMessage());
+                delegate.onRevisionSelected(selectionModel.getSelectedObject());
+            }
+        });
+        revisions.setSelectionModel(selectionModel);
+
+        revisions.addDomHandler(new DoubleClickHandler() {
+            @Override
+            public void onDoubleClick(DoubleClickEvent event) {
+                delegate.onRevisionDoubleClicked();
+            }
+        }, DoubleClickEvent.getType());
+
+        this.revisionsPanel.add(revisions);
+    }
+
+    private void createButtons() {
+        btnClose = createButton(locale.buttonClose(), "git-compare-branch-close", new ClickHandler() {
```

Comment:
at 89 line -> this.ensureDebugId("git-compare-revision-window");
But here "git-compare-branch"